README.md

```diff
@@ -1,81 +1,70 @@
+
+#### Setting up the Central Server
+Run the following to turn a TAO instance into a Central Server.
 
 ```php
 sudo -u www-data php index.php 'oat\taoOffline\scripts\tools\setup\SetupCentralServer'
 ```
-     In case taoEncryption it's installed this script it's gonna setup the instance with encryption.
 
-##### Types of Syncs available
-* Test Center Based on OrganisationId
+Again, instances with `taoEncryption` will benefit from encryption.
+
+#### Types of available synchrounizations
+* Test Center Based on the `OrganisationId`
     * Users:
         * Test Takers
         * Proctor Administrators
         * Proctor
         * Eligibility
         * Deliveries
- * Results   
- * Results Logs
-
-
-##### Overview Flow
-![alt text](docs/overview_sync.png)
-
-###### Sequence Diagram
-![alt text](docs/sync_flow.png)
-
-##### Sync Users with encryption
-Each users has assign to him the application id in order to have access to the delivery content.
-The properties that be excluded in the process of sync can be found in the `excludedProperties` in the config
-`config/taoSync/syncService.conf.php`
-In terms of encryption the properties that be encrypted are determined in `config/taoEncryption/encryptUserSyncFormatter.conf.php`
-
-
-![alt text](docs/sync_users.png)
+* Results   
+* Result Logs
 
-##### Sync Deliveries with encryption
-On each delivery sync the test package it's send to the client, the client it's importing the test and generating a delivery. 
-_Note_: 
-> In a case that we are syncing a delivery already existing on the VM a new import of the test will exist.
 
-![alt text](docs/sync_delivery.png)
+#### Overview of the workflow
+![Overview workflow](docs/overview_sync.png)
 
-###### Sync Results with encryption
-The results chunkSize it's a very important config that needs to be set in advance, by default it's 10. Based on the number of variables exists in a results this can be set.
-For example if you have a test of 100 items this will mean ~400 variables the total request will contain 4000 variables which will overload the server. In this case reducing the chunkSize to less it's adviced.
+##### Sequence Diagram
+![Sequence Diagram](docs/sync_flow.png)
 
-The statuses of a result which needs be sent can be configurable in the same config `statusExecutionsToSync`
-The config can be found in `config/taoSync/resultService.conf.php`
-Each request to the server will include the no of results, the process will stop after all results are sent.
+#### Synchronizing users with encryption
+Every user has been assigned an `application ID` that is used to grant access to the delivery content. Properties that are excluded from the synchronization process can be found under `excludedProperties` in the configuration file `config/taoSync/syncService.conf.php`.  Properties that are encrypted are defined inside `config/taoEncryption/encryptUserSyncFormatter.conf.php`.
```

Comment:
This sentence is not completely correct. User is assigned Application Key which is used to decrypt delivery content.

Reply:
done